tests/test_axis.py

```diff
@@ -793,7 +792,7 @@ def test_getitem(self, ref, growth):
     def test_iter(self, ref, growth):
         Cat = bh.axis.StrCategory if isinstance(ref[0], str) else bh.axis.IntCategory
         a = Cat(ref, growth=growth)
-        assert_array_equal(a, ref)
+        assert a == approx(ref)
```

Comment:
```suggestion
        assert np.asarray(a) == approx(ref)
```

Quick fix. But is it possible to control how pytest treats approx?